src/ContainerCommitModal.jsx

```diff
@@ -112,113 +93,73 @@ class ContainerCommitModal extends React.Component {
-            this.state.onbuild.map((bud, idx) => (
-                <Flex key={"onbuildvar" + idx} id="select-claimed-onbuildvars" className="containers-run-onbuildvarclaim">
-                    <TextInput value={this.state.onbuildvar_key} onChange={value => this.handleOnBuildsInputChange(idx, value)} />
-                    <FlexItem align={{ default: 'alignRight' }}>
-                        <Button variant="secondary" isDisabled={idx === 0}
-                                aria-label={_("Remove on build variable")}
-                                icon={<CloseIcon />}
-                                isSmall
-                                onClick={() => this.handleRemoveOnBuild(idx)} />
-                        <Button variant="secondary" onClick={this.handleAddOnBuild}
-                                icon={<PlusIcon />}
-                                isSmall
-                                aria-label={_("Add on build variable")} />
-                    </FlexItem>
-                </Flex>
-            ));
         const commitContent =
             <Form isHorizontal>
-                <FormGroup fieldId="commit-dialog-container-name" label={_("Container name")} hasNoPaddingTop>
-                    <span id="commit-dialog-container-name">
-                        {this.props.container.Names}
-                    </span>
-                </FormGroup>
-
-                <FormGroup fieldId="commit-dialog-format" label={_("Format")} isInline>
-                    <Radio id="format-oci" value="oci"
-                           name="format"
-                           label="oci"
-                           isChecked={this.state.selectedFormat === 'oci'}
-                           onChange={format => this.handleFormatChange(format)} />
-                    <Radio id="format-docker" value="docker"
-                           name="format"
-                           label="docker"
-                           isChecked={this.state.selectedFormat === 'docker'}
-                           onChange={format => this.handleFormatChange(format)} />
-                </FormGroup>
-
-                <FormGroup fieldId="commit-dialog-image-name" label={_("Image name")}>
+                <FormGroup fieldId="commit-dialog-image-name" label={_("New image name")}
+                           validated={this.state.nameError ? "error" : "default"}
+                           helperTextInvalid={this.state.nameError}>
                     <TextInput id="commit-dialog-image-name"
                                value={this.state.imageName}
+                               validated={this.state.nameError ? "error" : "default"}
                                onChange={value => this.handleInputChange("imageName", value)} />
                 </FormGroup>
 
                 <FormGroup fieldId="commit-dialog-image-tag" label={_("Tag")}>
                     <TextInput id="commit-dialog-image-tag"
+                               placeholder="latest" // Do not translate
```

Comment:
Ah, it's the default value, so I think we should just set it as value instead of a placeholder?

Reply:
@garrett I followed here the design, WDYT?

Reply:
"latest" is not an example; it is what you'd get by default if you don't enter anything.

https://www.patternfly.org/v4/components/form/design-guidelines/#placeholder-text-0

> If your placeholder text demonstrates a correct input, introduce it as an example so that users can distinguish impermanent examples from field labels and their own inputted text.

If we think it looks too inconsistent, then we can drop the latest and add some helper text saying that if no tag is given, then it defaults to "latest".

Reply:
What happens if author is blank?